src/main/java/net/minecraftforge/common/BiomeDictionary.java

```diff
@@ -96,6 +96,11 @@
         public static final Type BEACH = new Type("BEACH");
         public static final Type VOID = new Type("VOID");
 
+        /*Tags specifying the dimension a biome generates in. Specifying neither implies a biome that generates in a modded dimension*/
```

Comment:
This doesn't make any sense. Why use nether to mean two things? Just have the absence of any of these tags to mean "modded"

Reply:
Because right now the absence of these tags can mean either overworld or modded, no way to differentiate between these two. And there's a chance you probably you misread neither for nether.

Reply:
I did, yes. But the grammar is not correct there, as there are three options. It should be "none"